scipy/stats/_continuous_distns.py

```diff
@@ -5508,67 +5508,56 @@ def _entropy(self, s):
         this function uses explicit formulas for the maximum likelihood
         estimation of the log-normal shape and scale parameters, so the
         `optimizer`, `loc` and `scale` keyword arguments are ignored.
+        If the location is free, a likelihood maximum is found by
+        setting its partial derivative wrt to location to 0, and
+        solving by substituting the analytical expressions of shape
+        and scale (or provided parameters).
+        See, e.g., equation 3.1 in
+        A. Clifford Cohen & Betty Jones Whitten (1980)
```

Comment:
Can this be added to a [references](https://numpydoc.readthedocs.io/en/latest/format.html#references) section of the documentation from within the decorator? Then you can cite the reference.

Reply:
Done.